rss/parser.cpp

```diff
@@ -75,7 +81,10 @@ static size_t handle_headers(void* ptr, size_t size, size_t nmemb, void* data)
 	const auto header = std::string(reinterpret_cast<const char*>(ptr), size * nmemb);
 	HeaderValues* values = static_cast<HeaderValues*>(data);
 
-	if (header.find("Last-Modified:") == 0) {
+	if (header.find("HTTP") == 0) {
```

Comment:
Just an idea, but not sure if it were any better to check for `HTTP/` with a slash.

Reply:
I guess that might make it a lot less likely to incorrectly match on regular headers, so a good idea.
Thanks, done